.github/workflows/release.yml

```diff
@@ -35,6 +35,6 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         with:
           version: latest
-          args: release --rm-dist --release-notes=../release_notes.md
+          args: release --rm-dist
```

Comment:
just a note, this removes automatically adding messages to releases. After this point the releaser will have to manually edit the release.

Reply:
What messages do we usually add to our releases?

Reply:
links to the changelog https://github.com/tendermint/tendermint/blob/v0.34.12/CHANGELOG.md#v0.34.12.

the sdk just edits it, so we can do the same.

Reply:
I don't mind manually doing it - it's not too much work